classes/ActiveDesign.php

```diff
@@ -16,11 +16,6 @@ class ActiveDesign extends BaseActiveDesign
      */
     public function getId()
     {
-        // backwards compat for pre ProPhoto 6.14.0
```

Comment:
`isLiveDesign` - currently does not exist as a concept in P7 (though it will in some form). We can leave it for now and change it later if necessary?

Reply:
Did I see that you have an active designs PR open - shouldn't this concept of changed in that PR? I haven't looked at it, but I'm referring to https://github.com/downshiftorg/prophoto/pull/3035